### DISCUSSION TITLE
Removing supression from restricted account

### DISCUSSION THREAD

**Discussion by jmbm194** [codewars/codewars.com#3118] [Q&A]
I tested code that wasn't mine because i wanted to understand why my answers where failing and compare test cases in which other people's solutions passed. I wasn't aware this could lead to a ban because my priority isn't really accumulating points I just wanted to understand why my own solutions were failing. Can i please have my restriction removed?

account: jotabadme

**Comment by Blind4Basics** [comment]
Hello,

that definitely wasn't the only time this happened, so no, the account won't be unlocked. You can delete it and start fresh, on the other hand (without doing that again, ofc).

Cheers
